moe/plugins/__init__.py

```diff
@@ -0,0 +1 @@
+"""Official moe plugins."""
```

Comment:
Is it good practice to use the `__init__.py` file to just leave a docstring describing the module?

Reply:
Yes, I believe so, just to give some more information on the package. Are you saying as opposed to no docstring at all or a more detailed one?

Reply:
I just haven't seen `__init__.py` used this way (not that I'm an experienced Python programmer) so I was just curious. I think it's generally a good idea!